crypto/bio/bf_prefix.c

```diff
@@ -100,13 +107,25 @@ static int prefix_write(BIO *b, const char *out, size_t outl,
         size_t i;
         char c;
 
-        /* If we know that we're at the start of the line, output the prefix */
+        /*
+         * If we know that we're at the start of the line, output prefix and
+         * intendation.
+         */
         if (ctx->linestart) {
             size_t dontcare;
 
-            if (!BIO_write_ex(BIO_next(b), ctx->prefix, strlen(ctx->prefix),
-                              &dontcare))
+            if (ctx->prefix != NULL
+                && !BIO_write_ex(BIO_next(b), ctx->prefix, strlen(ctx->prefix),
+                                 &dontcare))
                 return 0;
+            /* Do initial indentation in chunks of 8, for efficiency */
+            for (i = 0; i + 8 <= ctx->indent; i += 8)
```

Comment:
Can't you do BIO_printf with a field width?

Reply:
I totally forgot!

Reply:
Done